advanced/am-pattern-1/README.md

```diff
@@ -1,4 +1,4 @@
-# Helm Chart for deployment of WSO2 API Manager with WSO2 API Manager Analytics
+# Helm Chart for deployment of WSO2 API Manager Pattern 1: Single node (all-in-one) deployment
```

Comment:
@Chathuru please include details about WSO2 product Docker image types in the README documentation. Please restructure the README as per [1]. Feel free to drop in your suggestions and concerns.

[1]: [IAM pattern 1 README documentation](https://github.com/wso2/kubernetes-is/blob/v5.10.0.1/advanced/is-pattern-1/README.md)